### DISCUSSION TITLE
Tabby appears to use a *LOT* of GPU ram, is this normal?

### DISCUSSION THREAD

**Discussion by jafin** [Eugeny/tabby#5667] [Q&A]
I was just investigating why I am using a large portion of GPU Ram,  I have a 1060 GB card with 6GB of VRAM.
Anyhow Checking "Dedicated GPU memory" in Task Manager showed me that Tabby was the second largest consumer (first being DWM - Desktop Window manager) taking 740MB of GPU Ram.

That seems like a LOT!
I have 3 `cmd` tabs open in tabby

For comparison, I opened `windows terminal` with 3 tabs, and its GPU RAM usage is 32MB

As a workaround, If I disabled GPU acceleration, it drops back 0 GPU memory usage.  

My point though, is even with GPU acceleration on, should it really be using that much VRAM?

**Comment by cui3093** [comment]
![image](https://user-images.githubusercontent.com/33884456/160062432-3cd1db95-4e44-4f15-8a12-b2fbf171bcef.png)
Two much memory consumption.

**Comment by zod076** [comment]
Just installed it and created 3 ssh profiles and it's memory usage is already at 400MB+ ..and after 3yrs this is still an issue.. Any way to reduce the memory consumption ?

**Reply by louishust** [reply]
4 year now
